app/controllers/providerList.js

```diff
@@ -0,0 +1,22 @@
+
+
+function info(){
+	
+var alertDialog = Titanium.UI.createAlertDialog({
```

Comment:
Since this is just going to be an info dialog, we could get away with just having a simple alert:
`alert('Connect with agencies to learn what programs...`);`